transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollStreamChannel.java

```diff
@@ -88,6 +75,7 @@
 
     private WritableByteChannel byteChannel;
 
+    @Deprecated
```

Comment:
I am not strong on this... just noticed we not use it at all so I saw no reason to not deprecate it. Let me remove the annotation for now